package.json

```diff
@@ -6,15 +6,11 @@
   "dependencies": {
     "nan": "~1.5.0"
   },
-  "os": [
-    "darwin"
```

Comment:
@aredridel ping

Aside from clearing this up, I think your patch will be good to merge